ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__automated-release-stable-4.19-upgrade-from-stable-4.19.yaml

```diff
@@ -0,0 +1,107 @@
+base_images:
+  ansible:
+    name: "4.19"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.19"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  openstack-installer:
+    name: "4.19"
+    namespace: ocp
+    tag: openstack-installer
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.19"
+  tests-private-preupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.19"
+  tools:
+    name: "4.19"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.19"
+    namespace: ocp
+    tag: upi-installer
+  verification-tests:
+    name: verification-tests
+    namespace: ci
+    tag: latest
+  vsphere-ci-python:
+    name: vsphere-python
+    namespace: ci
+    tag: latest
+releases:
+  latest:
+    prerelease:
+      architecture: amd64
```

Comment:
same concern

Reply:
done

Reply:
I think we need to add `relative: 1` here to get previous ec build for 4.19
xref: https://docs.ci.openshift.org/docs/architecture/ci-operator/#testing-with-an-existing-openshift-release

Reply:
Sure will do that